src/webapp/src/settings/settings.service.ts

```diff
@@ -164,66 +161,193 @@ export class SettingsService extends BaseService<SettingsEntity> {
     return '/'
   }
 
-  async getAssetsRoot (): Promise<any> {
-    const themeRoot = this.req.tenantId ?? 'default'
-    const assetsRoot = `/${themeRoot as string}`
+  async getAssetsRoot (): Promise<{themeRoot: string, assetsRoot: string}> {
+    const themeRoot: string = this.req.tenantId ?? 'default'
+    const assetsRoot: string = `/${themeRoot}`
     return { themeRoot, assetsRoot }
   }
 
   async getWebsiteRenderingVariables (): Promise<any> {
     const { themeRoot, assetsRoot } = await this.getAssetsRoot()
     const settings = await this.getWebsiteSettings()
 
-    const name = htmlEncode(settings.name)
-    const title = htmlEncode(settings.title)
-    const nameAndTitle = `${name} - ${title}`
-    const description = htmlEncode(settings.description)
-    const prelaunchEnabled = !!settings.prelaunch_enabled
-    const prelaunchHidePoweredby = !!settings.prelaunch_hide_poweredby || (this.req.headers != null && this.req.headers['x-tenant-name'] === 'saasform')
-    const prelaunchCleartextPassword = htmlEncode(settings.prelaunch_cleartext_password)
-    const prelaunchMessage = htmlEncode(settings.prelaunch_message)
-    const prelaunchBackground = htmlEncode(settings.prelaunch_background)
-    const googleAnalytics = htmlEncode(settings.google_analytics)
-    const googleTagManager = htmlEncode(settings.google_tag_manager)
-    const facebookPixelId = htmlEncode(settings.facebook_pixel_id)
-    const legalName = htmlEncode(settings.legal_name)
-    const domainPrimary = htmlEncode(settings.domain_primary)
-    const email = htmlEncode(settings.email)
-
-    const googleTagManagerHeader = googleTagManager != null
+    const htmlAsset: (string) => string = (asset: string) => (`${assetsRoot}/${asset}`)
+
+    // TODO make better/with types
+    const res = {
```

Comment:
I'm not in love with this solution, however I wanted a single place where we can see the final structure with all the variables. It's already getting ugly, and without a single place we'll never noticed it.